### DISCUSSION TITLE
middleware/redis/best-practice/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4786] [Announcements]
# middleware/redis/best-practice/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/redis/best-practice/

**Comment by ananleledad** [comment]
可以使用redis-GunYu这个工具做实时同步、数据迁移、备份等。
功能比较完善。
https://github.com/mgtv-tech/redis-GunYu

**Reply by lday0321** [reply]
有功能介绍的详细文档吗？
